conans/client/conf/__init__.py

```diff
@@ -84,6 +84,7 @@
 default_profile = %s
 compression_level = 9                 # environment CONAN_COMPRESSION_LEVEL
 sysrequires_sudo = True               # environment CONAN_SYSREQUIRES_SUDO
+request_timeout = 30                  # environment CONAN_REQUEST_TIMEOUT (seconds)
```

Comment:
I think introducing a 30 seconds timeout can be breaking, though not sure about the definition of timeout for requests lib. What happens to long transfers? Maybe the default timeout should be the current one (None)?

Reply:
Now I see this comment: https://github.com/conan-io/conan/issues/2716#issuecomment-379160290

Though it makes sense to have 30 seconds, it might be a bit short for some people that might be querying bintray repos, and the response might sometimes be slower than 30 seconds.